app/inputhelp.cpp

```diff
@@ -116,6 +116,11 @@ QString InputHelp::merginSubscriptionLink() const
   }
 }
 
+QString InputHelp::merginPricingLink() const
+{
+  return inputWeb + "/pricing" + utmTagSubscription;
+}
```

Comment:
I do not think we'll need this link

Reply:
Removed the need for support subscription and link to pricing page